### ISSUE TITLE
Graph cut segmentation disabled

### ISSUE THREAD

**Issue by akreshuk** [ilastik/ilastik#1061]
in the current binary (1.1.3), linux.

**Comment by burgerdev** [comment]
I tried this on a clean Ubuntu 14.10 virtual machine, seems like cplex is not optional any more...

```bash
burger@vbox:~/testing $ ilastik-1.1.3-Linux/bin/python -c 'import opengm'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/burger/testing/ilastik-1.1.3-Linux/lib/python2.7/site-packages/opengm/__init__.py", line 12, in <module>
    from _inference_interface_generator import _inject_interface , InferenceBase
  File "/home/burger/testing/ilastik-1.1.3-Linux/lib/python2.7/site-packages/opengm/_inference_interface_generator.py", line 8, in <module>
    from _inference_injector import _injectGenericInferenceInterface
  File "/home/burger/testing/ilastik-1.1.3-Linux/lib/python2.7/site-packages/opengm/_inference_injector.py", line 3, in <module>
    from inference  import InferenceTermination
  File "/home/burger/testing/ilastik-1.1.3-Linux/lib/python2.7/site-packages/opengm/inference/__init__.py", line 1, in <module>
    from _inference import *
ImportError: libilocplex.so: cannot open shared object file: No such file or directory
```

**Comment by burgerdev** [comment]
@stuartberg: Is it possible that we need a binary compiled for use with cplex and another one compiled without it?

**Comment by akreshuk** [comment]
You mean, not optional for opengm? That shouldn't really be so,  but maybe we accidentally built the opengm binary this way.

**Comment by stuarteberg** [comment]
Fixed by @chaubold via #1126
